modules/dicom_archive/jsx/dicom_archive.js

```diff
@@ -128,7 +128,8 @@ class DicomArchive extends Component {
       }},
       {label: 'Sex', show: true, filter: {
         name: 'sex',
-        type: 'text',
+        type: 'select',
+        options: options.sex.reduce((a, b)=> (a[b]=b, a), {}),
```

Comment:
This should not be coming from PHP. The values should just be coded in the React element.

Reply:
It feels more future-proof to do this on the front-end though. The values could change, why not do it in a functional way instead of hardcoding?

Reply:
@johnsaigle @driusan oh you want the array just made in the frontend. What if the database changes the M, F, O in the future to be Male, Female, Other. I think some of the backend should be sent over?

Reply:
The dicom standard specifies that the valid values are M, F, and O. If there's a chance that they might change, it should remain a text field and this PR should just be closed.

Reply:
@driusan ah if the case is never going to change then just a frontend change is cleaner.